api/app/hfi/pdf_generator.py

```diff
@@ -3,16 +3,42 @@
 import pdfkit
 from jinja2 import Environment, FunctionLoader
 from app.schemas.hfi_calc import FireCentre, HFIResultResponse, PlanningArea, WeatherStation
-from app.hfi.daily_template import str_daily_template, CSS_PATH
-from app.hfi.pdf_data_formatter import response_2_daily_jinja_format
+from app.hfi.pdf_template import PDFTemplateName, get_template, CSS_PATH
+from app.hfi.pdf_data_formatter import response_2_daily_jinja_format, response_2_prep_cycle_jinja_format
 
 # Loads template as string from a function
 # See: https://jinja.palletsprojects.com/en/3.0.x/api/?highlight=functionloader#jinja2.FunctionLoader
-jinja_env = Environment(loader=FunctionLoader(str_daily_template), autoescape=True)
+jinja_env = Environment(loader=FunctionLoader(get_template), autoescape=True)
 
 
-def generate_daily_pdf(result: HFIResultResponse, fire_centres: List[FireCentre]) -> bytes:
-    """Generate a daily PDF"""
+def generate_pdf(result: HFIResultResponse, fire_centres: List[FireCentre]) -> bytes:
+    """Generates the full PDF based on the HFIResultResponse"""
+
+    rendered_output: str = ''
```

Comment:
Isn't this line redundant? Could you just start with
`rendered_output = generate_prep(result)`